packages/node_modules/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -0,0 +1,322 @@
+import uuid from 'uuid';
+import {EVENT_TRIGGERS, VOICEA_RELAY_TYPES} from '@webex/internal-plugin-voicea/src/constants';
+import LLMChannel from '@webex/internal-plugin-llm/src/index';
+
+import Trigger from '../../plugin-meetings/src/common/events/trigger-proxy.js';
+
+// eslint-disable-next-line no-unused-vars
+import {AnnouncementPayload, CaptionLanguageResponse, TranscriptionResponse} from './voicea.types';
+
+
+export const config = {
```

Comment:
Do we need a config here, Should only be in LLM

Reply:
removed